libminifi/include/utils/file/FileUtils.h

```diff
@@ -79,16 +82,14 @@ class FileUtils {
 
   FileUtils() = delete;
 
-  /**
-   * Deletes a directory, deleting recursively if delete_files_recursively is true
-   * @param path current path to delete
-   * @param delete_files_recursively deletes recursively
-   */
-
-  static char get_separator()
+  static char get_separator(bool force_posix = false)
```

Comment:
Can a comment clarify the purpose of force posix?

Reply:
I hope it does now.